OpenOversight/app/main/model_view.py

```diff
@@ -77,11 +77,16 @@ def new(self, form=None):
                 add_department_query(form, current_user)
                 if getattr(current_user, "dept_pref_rel", None):
                     set_dynamic_default(form.department, current_user.dept_pref_rel)
+
+            now = datetime.datetime.now()
             if hasattr(form, "created_by") and not form.created_by.data:
                 form.created_by.data = current_user.get_id()
+            if hasattr(form, "created_at") and not form.created_at.data:
+                # TODO: Add created_at field to forms
+                form.created_at.data = now
             if hasattr(form, "last_updated_by"):
                 form.last_updated_by.data = current_user.get_id()
-                form.last_updated_at.data = datetime.datetime.now()
+                form.last_updated_at.data = now
```

Comment:
Should `created_by` and `last_updated_by` be implemented as hidden form fields? Users are able to arbitrarily modify the fields using devtools

Reply:
They shouldn't be, nope! I'm removing those in a PR now.